frontends/web/src/routes/buy/info.tsx

```diff
@@ -151,12 +151,12 @@ class BuyInfo extends Component<Props, State> {
                                                 </tr>
                                                 <tr>
                                                     <td>{t('buy.info.disclaimer.payment.table.2_method')}</td>
```

Comment:
I have to say it's not immediately clear which payment method this is by looking at the code... A better name would've been `cc_method`, `mobile_method`, `bank_method`, etc.

Reply:
Sorry I only saw LGTM and merged...

Reply:
No worries, it was for the future. Remember next time :)